CLAUDE.md

```
@@ -112,8 +112,8 @@ Skills are located in subdirectories of this plugin. Read the relevant `SKILL.md
 |-------|-------------|
 | `refactor-code` | Restructuring code while preserving behavior |
 
-### Workflow Skills (Orchestrators)
-| Skill | Use when... |
+### Primary Agents
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)